viewer/packages/pointclouds/src/potree-three-loader/rendering/constants.ts

```diff
@@ -14,5 +14,5 @@ export const PERSPECTIVE_CAMERA = 'PerspectiveCamera';
 export const COLOR_BLACK = new Color(0, 0, 0);
 export const COLOR_WHITE = new Color(1, 1, 1);
 export const DEFAULT_HIGHLIGHT_COLOR = new Vector4(1, 0, 0, 1);
-export const OBJECT_STYLING_TEXTURE_WIDTH = 1024;
-export const OBJECT_STYLING_TEXTURE_HEIGHT = 1024;
+export const OBJECT_STYLING_TEXTURE_WIDTH = 256;
+export const OBJECT_STYLING_TEXTURE_HEIGHT = 256;
```

Comment:
Hmm, why is this constant? Doesn't this enforce a limit on 65K objects?

Reply:
Yes, this enforces a limit. I can make it higher, but we should probably agree on a reasonable limit. Not sure how easy it would be to make it dynamic.